de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/joiner/JoinerNodeModel.java

```diff
@@ -844,21 +852,42 @@ private static String getObjectAsJSONString(Object object) {
     return new String[] {getObjectAsJSONString(portObject.modelMetadata),
         getObjectAsJSONString(portObject.getModel()), getObjectAsJSONString(portObject.getViz()),
         getObjectAsJSONString(portObject.simulations), getObjectAsJSONString(portObject.packages),
-        ""};
+        "", ""};
+  }
+
+  @SuppressWarnings("resource")
```

Comment:
There is a potential resource leak here as the stream may not be closed on IO errors. This warning should not be suppressed. Streams can be closed automatically in case of errors with try-with resources.